tests/by-util/test_cp.rs

```diff
@@ -505,9 +505,30 @@ fn test_cp_arg_interactive_update() {
     at.touch("a");
     at.touch("b");
     ucmd.args(&["-i", "-u", "a", "b"])
-        .pipe_in("N\n")
+        .pipe_in("")
         .succeeds()
         .no_stdout();
```

Comment:
Why not remove these lines (plus line 504 (Github doesn't allow me to select it))? No prompt is expected in this scenario and so it is unnecessary to call `pipe_in` at all.

Reply:
Good question, that pattern was used in a few other places, so I blindly followed it.

Now that I think about it, it seems unnecessary. I'll remove it, thanks!

Reply:
Ah, I remember! It was to make absolutely extra-sure that piping-in nothing and closing stdin behaves identically, that's why there are two calls in `test_cp_arg_interactive_update`. I'll add a comment instead.